test_env_vars.yaml

```diff
@@ -220,123 +222,102 @@ gomod_packages:
-        version: "v0.0.0-20170915032832-14c0d48ead0c"
-      - name: "golang.org/x/text/internal/tag"
-        replaces: null
-        type: "go-package"
-        version: "v0.0.0-20170915032832-14c0d48ead0c"
-      - name: "rsc.io/sampler"
-        replaces: null
-        type: "gomod"
-        version: "v1.3.0"
-      - name: "rsc.io/quote"
-        replaces: null
-        type: "gomod"
-        version: "v1.5.2"
-      - name: "golang.org/x/text"
-        replaces: null
-        type: "gomod"
-        version: "v0.0.0-20170915032832-14c0d48ead0c"
+    response_parts_to_check:
+      packages:
+        - dependencies:
+            - name: "rsc.io/sampler"
+              replaces: null
+              type: "gomod"
+              version: "v1.3.0"
+            - name: "rsc.io/quote"
+              replaces: null
+              type: "gomod"
+              version: "v1.5.2"
+            - name: "golang.org/x/text"
+              replaces: null
+              type: "gomod"
+              version: "v0.0.0-20170915032832-14c0d48ead0c"
+          name: "github.com/cachito-testing/cachito-gomod-with-deps"
+          type: "gomod"
+          version: "v0.0.0-20201001160613-4c65d49cae6b"
+        - dependencies:
+            - name: "rsc.io/quote"
+              replaces: null
+              type: "go-package"
+              version: "v1.5.2"
+            - name: "rsc.io/sampler"
+              replaces: null
+              type: "go-package"
+              version: "v1.3.0"
+            - name: "golang.org/x/text/language"
+              replaces: null
+              type: "go-package"
+              version: "v0.0.0-20170915032832-14c0d48ead0c"
+            - name: "golang.org/x/text/internal/tag"
+              replaces: null
+              type: "go-package"
+              version: "v0.0.0-20170915032832-14c0d48ead0c"
+          name: "github.com/cachito-testing/cachito-gomod-with-deps"
+          type: "go-package"
+          version: "v0.0.0-20201001160613-4c65d49cae6b"
+      dependencies:
+        - name: "rsc.io/quote"
+          replaces: null
+          type: "go-package"
+          version: "v1.5.2"
+        - name: "rsc.io/sampler"
+          replaces: null
+          type: "go-package"
+          version: "v1.3.0"
+        - name: "golang.org/x/text/language"
+          replaces: null
+          type: "go-package"
+          version: "v0.0.0-20170915032832-14c0d48ead0c"
+        - name: "golang.org/x/text/internal/tag"
+          replaces: null
+          type: "go-package"
+          version: "v0.0.0-20170915032832-14c0d48ead0c"
+        - name: "rsc.io/sampler"
+          replaces: null
+          type: "gomod"
+          version: "v1.3.0"
+        - name: "rsc.io/quote"
+          replaces: null
+          type: "gomod"
+          version: "v1.5.2"
+        - name: "golang.org/x/text"
+          replaces: null
+          type: "gomod"
+          version: "v0.0.0-20170915032832-14c0d48ead0c"
+    files_to_ignore: ["modvendor"]
```

Comment:
Do we need this here? It would be nice to check how this file is being generated.